### PR TITLE
Some new flaties

### PR DESCRIPTION
**Pull request by fritzmark**
Also adjusted actions since they caused some regression errors.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -140,13 +145,15 @@ Gerne weitere als Pull-Request hinzufügen.
...
 - Was ist das agressivste Milchprodukt? Schlagsahne.
+- Was ist das zerstörerischste Tier? Der Schmetterling.
 - Was ist der schlauste Ast? Ein Gymniasast.
 - Was ist der unförmigste Seevogel von Nordfriesland? Die Amöve.
 - Was ist die größte Anerkennung unter Aquarellisten? "Verläuft bei Dir!"
 - Was ist durchsichtig, stinkt und ignorant? Ein Schnurz.
 - Was ist ein Cowboy ohne Pferd? Ein Sattelschlepper.
 - Was ist ein Keks unter einem Baum? Ein schattiges Plätzchen.
 - Was ist ein nackter Ritter? Ent-rüstet!
+- Was ist eine Bombe am liebsten? KartoffelPUFFer.
```
Comment (ankon) [inline README.md:156]: *isst?

**Comment by derphilipp** [comment]
Thank you for your valuable work!

File: README.md (hunk 1 above)
Comment (fritzmark) [reply README.md:156]: Right! Will adapt it.

**Comment by fritzmark** [comment]
Whoop. Let the bellies explode.

**Pull request closed without merging**